apps/react/src/features/studios/components/AddStudioButton/AddStudioButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, memo } from 'react';
+import { IconButton, Typography } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
```

Comment:
Please, prefer to use named export/import instead of default.
Usually it's better for code navigation and in some cases it might reduce bundle size.

```suggestion
import { AddIcon } from '@mui/icons-material';
```

To dive deeper into the topic you can read [this](https://www.freecodecamp.org/news/difference-between-default-and-named-exports-in-javascript/) and [this](https://dev.to/phuocng/avoid-using-default-exports-a1c) articles.

Reply:
Please, check similar code in all components. Try to fix it everywhere.